kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/certificate/enrollment/EnrollmentService.java

```diff
@@ -36,29 +37,38 @@ public interface EnrollmentService {
     public void enroll() throws KuraException;
 
     /**
-     * Renews the client certificate creating a new keypair and a new CSR submitted to the CA.
+     * Renew the client certificate submitting the certificate previously issued by the CA.
      * 
      * @throws KuraException
-     *             it it is impossible to renew the system certificate
+     *             if it is impossible to renew the system certificate
      */
     public void renew() throws KuraException;
 
     /**
+     * Renew the client certificate creating a new keypair and a new CSR submitted to the CA.
      * 
-     * Get the Certificate Authority <code>Certificate</code> stored in the relative
+     * @throws KuraException
+     *             if it is impossible to renew the system certificate
+     */
+    public void rekey() throws KuraException;
+
+    /**
+     * 
+     * Get the list of the Certificate Authority certificates stored in the relative
      * {@link org.eclipse.kura.security.keystore.KeystoreService}
      * 
-     * @return returns the Certificate Authority <code>Certificate</code>
+     * @return returns the Certificate Authority <code>CertStore</code>
      * @throws KuraException
-     *             if it is impossible to retrieve the certificate.
+     *             if it is impossible to retrieve the certificate store.
      */
-    public Certificate getCACertificate() throws KuraException;
+    public CertStore getCACertificate() throws KuraException;
```

Comment:
Since the CertStore contains multiple certificates, does it make sense to use getCACertificate**s** instead?